File: program_data/github_java_sort_function/6/98.java
@Override
    public void sort(Integer[] arr, Integer sortOrder) {
        long startTime = System.nanoTime();
        System.out.println("Before Sorting");
        SortingHelper.printArray(arr);
        int[] countArr = new int[10]; 
        int max = Collections.max(Arrays.asList(arr));
        Integer[] aux = new Integer[arr.length];

        int passes = 1;
        while (max > 0) { 

            for (Integer num : arr) {
                int index = (num % (passes * 10)) / passes;
                countArr[index]++;
            }

            countArr[0]--;
            
            for (int i = 1; i < 10; i++) {
                countArr[i] = countArr[i] + countArr[i - 1];
            }

            for (int i = arr.length - 1; i >= 0; i--) {
                int index = (arr[i] % (passes * 10)) / passes;
                int value = countArr[index];
                aux[value] = arr[i];
                countArr[index]--;
            }
            arr = Arrays.copyOf(aux, arr.length);
            Arrays.fill(countArr, 0);
            passes = passes * 10;
            max = max / 10;
        }

        System.out.println("After Sorting");
        SortingHelper.printArray(arr);
        SortingHelper.printStats(startTime);
    }
}

File: program_data/PKU_raw/82/98.c
int main()
{
	int n,arr[100][2],i,j=0,h[100]={0},m;
	cin>>n;
	for(i=0;i<n;i++)
	{
		cin>>arr[i][0]>>arr[i][1];
	}
	for(i=0;i<n;i++)
	{
		if(arr[i][0]<=140&&arr[i][0]>=90&&arr[i][1]<=90&&arr[i][1]>=60)
		{
			h[j]=h[j]+1;
		}
		else
		{
			j++;
		}
	}
	m=h[0];
	for(i=1;i<100;i++)
	{
		if(h[i]>m)
		{
			m=h[i];
		}
	}
	cout<<m;
	return 0;
}